### PR TITLE
Add initial draft of intro to envs chapter

### PR DESCRIPTION
**Pull request by jni**
CC @NelleV

### CODE REVIEW

**Comment by NelleV** [comment]
I think I don't have write access to the repository. Else, I would merge :)

**Comment by jni** [comment]
@NelleV are you saying that my prose is perfect? =P

**Comment by jni** [comment]
(invited you to maintain)

File: using-python-for-science/intro-to-environments.md [hunk 1]
```diff
@@ -0,0 +1,322 @@
...
+make: we can either install miniconda or Anaconda. Miniconda is *just* conda,
+and what we will install going forward. Anaconda includes miniconda *as well
+as* a whole bunch of Scientific Python packages hand-compiled by the engineers
+at Anaconda, Inc, all neatly bundled into a single, massive environment.
+Although this can be convenient, in our case we want to create, modify, and
+destroy environments at will, so we will skip the One Huge Environment To Rule
+Them All.
+
+As of this writing, you can download miniconda at
+https://anaconda.org/miniconda, but if that changes, simply search the web for
```
Comment (jni) [inline using-python-for-science/intro-to-environments.md:90]: This URL is actually totally wrong :joy: I'm wondering whether we should point users to miniforge instead? https://github.com/conda-forge/miniforge

Or micromamba: https://mamba.readthedocs.io/en/latest/micromamba.html  <-- but I'm more hesitant about this, much more recent.

**Pull request merged**